---
title: "Bioinformatics and Digital Epidemiology"
---

### Background and Rationale {#background}

An outbreak of an infectious disease can have a tremendous effect on people's health, national economy, and social wellbeing, thus early discovery of an infectious disease allows for local epidemic containment, reducing negative public health impact.

Traditional outbreak detection of public health important organisms through the demographic review approach coupled with diagnostic information heavily rely on the phenotypic features of the organism.  This technique has limited resolution required to map the source of the outbreak and to demonstrate chains of transmission, since isolates were grouped together as “indistinguishable” members of the cluster. Advancements in molecular biology such as whole-genome sequencing (WGS) overcome this limitation by allowing for identification of all genomic characteristics of the bacteria included in a cluster.

The Antimicrobial Resistance Surveillance Reference Laboratory developed and is currently implementing the SaTScan and WGS enhanced outbreak investigation protocol. The reference laboratory analyzes the AMR surveillance data from its sentinel sites for clustering through SaTScan module of WHONET software(laboratory information management system). SaTScan is a free cluster-detection software application within WHONET used by epidemiologists globally to describe spatial clusters of infectious and chronic disease. The isolates from detected clusters are subsequently sequenced to provide evidence to inform the outbreak investigation.

In an outbreak investigation, the availability and completeness of clinical-epidemiological and genomic data are important to determine the potential source and trace the transmission of the outbreak. Today, there are various digital tools available to help data integration and visualization of surveillance data especially during outbreak investigation. The Centre for Genomic Pathogen Surveillance (CGPS) at the University of Oxford’s Big Data Institute developed a number of web-based tools for data collection (Epicollect), bioinformatics analysis (Pathogenwatch), data integration (Dataflo) and visualization (Microreact).

The conduct of workshop/trainings aims to provide an avenue to showcase the contributions of the Philippine Antimicrobial Resistance Program to the country’s fight against AMR and to highlight the system data architecture of the program. Furthermore, this workshop plans to introduce the reference laboratory’s service of SaTScan and WGS guided outbreak investigation. Moreover, to promote awareness of various digital epidemiology tools for surveillance and outbreak investigation. It likewise aims to foster engagement of the relevant units of the Research Institute for Tropical Medicine towards the significance of a data management, system architecture and use of digital epidemiology tools.
